go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/microsoft/moc
 
-go 1.14
+go 1.15
 
```

Comment:
pipeline.yaml changes?

Reply:
The moc/moc-pkg pipelines don't currently have the GoTool task in their build steps. Should I add that as part of this PR?

Reply:
Yes, lets add it

Reply:
added GoTool task